### PR TITLE
Replace participation <select> filter by dropdown

### PR DESCRIPTION
**Pull request by kleingeist**
`<select>`s are not styled correctly.

### CODE REVIEW

File: meinberlin/apps/plans/assets/plans_map.jsx [hunk 1]
```diff
@@ -321,10 +325,38 @@ class PlansMap extends React.Component {
...
+              </button>
+              <ul aria-labelledby="id_filter_participation" className="dropdown-menu">
+                <li>
+                  <button
+                    type="button"
+                    className="dropdown-item select-item"
+                    value="-1"
+                    onClick={this.onParticipationFilterChange.bind(this)}>
+                    {django.gettext('All')}
+                  </button>
```
Comment (xi) [inline meinberlin/apps/plans/assets/plans_map.jsx]: Is there a specific reason why you used the `select-item` class here? There is no icon, so I see no real reason.

Comment (kleingeist) [reply meinberlin/apps/plans/assets/plans_map.jsx]: oh this has been probably a copy and paste error. will fix it.

**Comment by kleingeist** [comment]
for reference a link to the draft DrowDown react component 
https://github.com/liqd/a4-meinberlin/compare/2017-11-jd-react-dropdown

**Comment by kleingeist** [comment]
@xi i updated the PR  in regard to your feedback

**Pull request merged**